packages/gravity/kube-testing/src/spec/transport.ts

```diff
@@ -192,7 +106,15 @@ export class TransportTestPlan implements TestPlan<TransportTestSpec, TransportA
 
         await Promise.all(
           swarms.map(async (swarm, swarmIdx) => {
```

Comment:
not sure if having multiple swarms is useful in this test, how is it different from just adding more feeds?